.github/actions/setup-build/action.yml

```diff
@@ -40,6 +40,15 @@ runs:
       key: ${{ runner.os }}-stablehlo_build_assets-${{ inputs.llvm-version }}
       max-size: 4G
 
+  # Setup Bazelisk and caching using GitHub cache
+  - name: Setup Bazelisk
+    uses: bazelbuild/setup-bazelisk@v2
+  - name: Mount bazel cache
+    uses: actions/cache@v3
+    with:
+      path: "~/.cache/bazel"
+      key: bazel
```

Comment:
Not entirely sure, but does key need to be unique per cached thing? I.e. if we bump LLVM revision will bazel cache silently pass without updating?

May need to make key based on llvm-version as above (see line 40)